static/css/style.css

```diff
@@ -245,8 +253,40 @@ html .all-auth-container {
 .list li{
     font-size: 0.5em;
     text-align: left;
+}
+
+/* View Hackathon Page */
+
+.page-title:after {
+    content: '';
+    display: block;
+    height: 2px;
+    width: 100%;
+    background: var(--p-orange);
+}
+
+#hackathon-signup {
+    text-align: left;
+    margin: 5px 0;
+}
 
+.hackathon-sub-details {
+    margin: 0.8rem 0 0;
 }
+
+.hackathon-sub-details:after {
```

Comment:
CSS3 standard is `::after` instead of `:after`.

Reply:
Changed as suggested.